fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -620,12 +620,12 @@ class AltimeterIndicator extends DisplayComponent<AltimeterIndicatorProps> {
 
     private handleBlink() {
         if (this.mode.get() === 'STD') {
-            if (this.flightPhase > 3 && this.transAltAppr > this.props.altitude.get() && this.transAltAppr !== 0) {
+            if (this.flightPhase > 3 && this.transLvlAr.isNormalOperation() && 100 * this.transLvlAr.value > this.props.altitude.get()) {
                 this.stdGroup.instance.classList.add('BlinkInfinite');
             } else {
                 this.stdGroup.instance.classList.remove('BlinkInfinite');
             }
-        } else if (this.flightPhase <= 3 && this.transAlt < this.props.altitude.get() && this.transAlt !== 0) {
+        } else if (this.flightPhase <= 3 && this.transAltAr.isNormalOperation() && this.transAltAr.value < this.props.altitude.get()) {
```

Comment:
``  100 * this.transLvlAr.value > this.props.altitude.get() `` vs ``this.transAltAr.value < this.props.altitude.get()``

Is there a reason one value is multiplied by 100 and in the other case it is not? It seems like a mistake

Reply:
also clearer variable naming might help i.e. is this altitude in ft or flight level

Reply:
> ` 100 * this.transLvlAr.value > this.props.altitude.get()` vs `this.transAltAr.value < this.props.altitude.get()`
> 
> Is there a reason one value is multiplied by 100 and in the other case it is not? It seems like a mistake

No, this is correct. One is a flight level while the other is an altitude.

Reply:
> also clearer variable naming might help i.e. is this altitude in ft or flight level

One is transAlt(itude) and the other is transLevel... what could be clearer?